packages/@okta/vuepress-site/docs/guides/implement-oauth-for-okta-serviceapp/main/index.md

```diff
@@ -55,6 +55,8 @@ Create an OAuth 2.0 service app integration using the Admin Console.
 
 4. Enter a name for your app integration and click **Save**.
 
+5. Optionally, click the **Application rate limits** tab to adjust the percentage rate limit capacity for this service application. By default, each new application has this percentage set at 50%.
```

Comment:
Should this be `Optional. `?

`percentage rate limit capacity` seems a little unclear.

`has this percentage set at` also seems a little informal. Maybe something like "sets the rate limit capacity to 50%"?